cnd/migrations/2020-04-09-030124_create-message-tables/up.sql

```diff
@@ -0,0 +1,110 @@
+-- Your SQL goes here
+
+
+CREATE TABLE rfc003_bitcoin_ethereum_bitcoin_ether_request_messages
```

Comment:
Why is this migration creating tables already created in a previous migration? I would expect the new migration to only create tables introduced with this patch.

Same for `down.sql`. The migration should only apply new changes, not re-create the whole DB from scratch.

If we apply this new migration to an existing DB, it'd fail as it'd try to create tables that already exist.

If we fallback this new migration to an existing DB, all data, even the one in the old table would be lost.

Reply:
Oh thanks, I clearly do not understand how migrations work :)  Will fix